microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,147 @@
+
+For example, if a snapshot of the DOM was recorded in the enhanced trace, the **Elements** tool displays this snapshot. If console messages were recorded, the **Console** tool prints these messages. The **Sources** tool displays the scripts that were present during the recording.
+
+Using enhanced traces makes it possible to reliably resolve source code references found in imported traces to the actual runtime code in the **Sources** tool. Additionally, if source maps were presents when a trace was recorded, it will also be possible to resolve code references to their original source code.
+
+Enhanced traces can only be imported in Microsoft Edge, while normal traces are compatible with other browsers based on the Chromium engine.
+
+
+<!-- ====================================================================== -->
+## Set the default trace type
+
+By default, the **Performance** and **Memory** tools export data as normal traces, but it's possible to choose which type of traces are exported.
+
+To change the trace type:
+
+1. Open DevTools by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).
+
+1. In DevTools, in the upper right, click **Settings** (![Settings button](../media/settings-button-icon.msft.png)).
+
+1. Under **Persistence**, if you want to record enhanced traces, check **Export Enhanced Performance and Memory Traces**. Otherwise clear the option.
+
+   ![The Settings panel in DevTools, with the enhanced traces option checked](./share-traces-images/trace-type-setting.png)
+
+1. Enhanced traces can also optionally include console messages, script sources and DOM elements too. Choose what to include in enhanced traces by using these options:
+
+   * **Include console message in enhanced traces**.
+   * **Include script sources in enhanced traces**.
+   * **Include DOM snapshots in enhanced traces (experimental)**.
+
+   ![The Settings panel in DevTools, showing the other enhanced trace options](./share-traces-images/enhanced-trace-options.png)
+
+
+<!-- ====================================================================== -->
+## Export a trace
+
+To export a performance recording from the **Performance** tool:
+
+1. Go to the website you want to record the performance of.
+
+1. Open the **Performance** tool.
+
+1. Press **Record**, execute the scenario you want to investigate the performance of by interacting with the webpage, and then press **Stop**.
+
+1. When the performance profile appears, click **Save profile** (![Save profile button](../media/export-icon.png)).
+
+1. Choose a location to save the trace file on your disk.
+
+   ![Microsoft Edge with DevTools open, the Save profile button was clicked in the Performance tool, and an Explorer window is opened as a result, to save the exported file](./share-traces-images/exporting-perf-trace.png)
```

Comment:
* Break png into two pngs.
* Remove red box from the Save As window png.
* Put this step above the Save As window png, w/ a colon to glue the flow:
1. Choose a location to save the trace file on your disk:

Reply:
I'm splitting the screenshots. I don't think adding the `.devtools` part is a good idea though. The article explains the different between normal and enhanced traces, but it's up to them to choose which type of trace they want. They may be exporting either of these types. The screenshot may show a devtools file, because that's what I chose when taking the screenshot, but the reader may be exporting a normal trace.